src/main/java/org/gridsuite/study/server/dto/LoadFlowParameters.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.study.server.dto;
+
+import com.datastax.driver.core.DataType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.cassandra.core.mapping.CassandraType;
+import org.springframework.data.cassandra.core.mapping.UserDefinedType;
+
+import java.io.Serializable;
+
+@UserDefinedType("loadFlowParameters")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class LoadFlowParameters implements Serializable {
```

Comment:
You don't need a DTO for LoadFlowParameter. Powsybl provides one with a versionned JSON serialization, so it could be used safely in a REST API.